site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -23,114 +23,179 @@ type: docs
 weight: 600
 ---
 
-The default `polaris-server.yml` configuration is intended for development and testing. When deploying Polaris in production, there are several best practices to keep in mind.
+## Tuning Polaris for Production
 
-## Security
+The default server configuration is intended for development and testing. When deploying Polaris in
+production, there are several best practices to keep in mind.
 
-### Configurations
+Notable configuration options used to secure a Polaris deployment are outlined below.
 
-Notable configuration used to secure a Polaris deployment are outlined below.
+For more information on how to configure Polaris and what configuration options are available,
+refer to the Configuration Reference page.
 
-#### oauth2
+### Security
 
-> [!WARNING]  
-> Ensure that the `tokenBroker` setting reflects the token broker specified in `authenticator` below.
+Notable configuration options used to secure a Polaris deployment are outlined below.
 
-* Configure [OAuth](https://oauth.net/2/) with this setting. Remove the `TestInlineBearerTokenPolarisAuthenticator` option and uncomment the `DefaultPolarisAuthenticator` authenticator option beneath it.
-* Then, configure the token broker. You can configure the token broker to use either [asymmetric](https://github.com/apache/polaris/blob/b482617bf8cc508b37dbedf3ebc81a9408160a5e/polaris-service/src/main/java/io/polaris/service/auth/JWTRSAKeyPair.java#L24) or [symmetric](https://github.com/apache/polaris/blob/b482617bf8cc508b37dbedf3ebc81a9408160a5e/polaris-service/src/main/java/io/polaris/service/auth/JWTSymmetricKeyBroker.java#L23) keys.
+Polaris authentication requires specifying a token broker factory type. Two types are supported:
 
-#### authenticator.tokenBroker
+- `rsa-key-pair` uses a pair of public and private keys;
+- `symmetric-key` uses a shared secret.
 
-> [!WARNING]  
-> Ensure that the `tokenBroker` setting reflects the token broker specified in `oauth2` above.
+By default, Polaris uses `rsa-key-pair`, with randomly generated keys. 
 
-#### callContextResolver & realmIdResolver
-* Use these configurations to specify a service that can resolve a realm from bearer tokens.
-* The service(s) used here must implement the relevant interfaces (i.e. [CallContextResolver](https://github.com/apache/polaris/blob/8290019c10290a600e40b35ddb1e2f54bf99e120/polaris-service/src/main/java/io/polaris/service/context/CallContextResolver.java#L27) and [RealmContextResolver](https://github.com/apache/polaris/blob/7ce86f10a68a3b56aed766235c88d6027c0de038/polaris-service/src/main/java/io/polaris/service/context/RealmContextResolver.java)).
+> [!WARNING]  
+> The default `rsa-key-pair` configuration is not suitable when deploying many replicas of Polaris,
```

Comment:
This is a great callout.

I do fear that the website doesn't render `[!WARNING]` blocks correctly though

Reply:
Indeed I don't know where I took this from 😅 Let me replace by bold characters.

Reply:
Hmm actually it seems that `[!IMPORTANT]` is supported, I will use that.